.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,100 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: To reproduce
+      description: "Steps to reproduce the behavior:"
+      placeholder: |
+        1.
+        2.
+        3.
+        ...
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      placeholder: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating system
+      description: On what operating system GLPI agent is experimenting this issue ?
```

Comment:
```suggestion
      description: On which operating system does the GLPI agent experimenting this issue ?
```

Reply:
The fist change is good, but the second one doesnt make sense to me. Using "which" instead of "what" is used when there are specific options given.